spring-amqp/src/main/java/org/springframework/amqp/core/MessageProperties.java

```diff
@@ -331,6 +331,30 @@ public void setConsumerQueue(String consumerQueue) {
 		this.consumerQueue = consumerQueue;
 	}
 
+	/**
+	 * The x-delay header.
+	 * @return the delay.
+	 * @since 1.6
+	 */
+	public Integer getXDelay() {
+		Object xDelay = this.headers.get("x-delay");
+		if (xDelay instanceof Integer) {
+			return (Integer) xDelay;
+		}
+		else {
+			return null;
+		}
+	}
+
+	/**
+	 * Set the x-delay header.
+	 * @param xDelay the delay.
+	 * @since 1.6
+	 */
+	public void setXDelay(Integer xDelay) {
```

Comment:
Even if the plugin becomes standard, it's still an extension to the spec but, yes, we can remove the X.